lighthouse-cli/run.ts

```diff
@@ -29,28 +29,23 @@ interface LighthouseError extends Error {
   code?: string
 }
 
-// Boolean values that are true are rendered without a value (--debug vs. --debug=true)
-function formatArg(arg: string, value: any): string {
-  if (value === true) {
-    return `--${arg}`
-  }
-
-  // Only quote the value if it contains spaces
-  if (value.indexOf(' ') !== -1) {
-    value = `"${value}"`
-  }
-
-  return `--${arg}=${value}`
-}
-
 // exported for testing
 export function parseChromeFlags(flags: string) {
-  let args = yargs(flags).argv;
-  const allKeys = Object.keys(args);
-
-  // Remove unneeded arguments (yargs includes a _ arg, $-prefixed args, and camelCase dupes)
-  return allKeys.filter(k => k !== '_' && !k.startsWith('$') && !/[A-Z]/.test(k))
-      .map(k => formatArg(k, args[k]));
+  const parsed = yargsParser(
+      flags || '', {configuration: {'camel-case-expansion': false, 'boolean-negation': false}});
+
+  const addQuotes = (str: String) =>
+      (typeof str === 'string' && str.includes(' ')) ? `"${str}"` : str;
+
+  return Object
+      .keys(parsed)
+      // Remove unnecessary _ item provided by yargs,
+      .filter(key => key !== '_')
+      // Avoid '=true', then reintroduce quotes if required
+      .map(key => {
+        if (parsed[key] === true) return `--${key}`;
+        return `--${key}=${addQuotes(parsed[key])}`;
```

Comment:
nit: not sure this is worth creating a function for anymore since it's not used anywhere else and only handles the one case now

aside: does it do any harm to quote the value if it doesn't contain spaces?

Reply:
> aside: does it do any harm to quote the value if it doesn't contain spaces?

it does yes. tested this out. i think.

Reply:
> nit: not sure this is worth creating a function for anymore

pshhhh... there was a ToTT about this before! 
I would prefer having a clear name for this simple line than mentally parse it each time i reread the source.

Reply:
perma-quoting now. :)